src/backends/stimulus_gpu.hpp

```diff
@@ -0,0 +1,149 @@
+#pragma once
+
+#include <cmath>
+#include <limits>
+
+#include <mechanism.hpp>
+#include <algorithms.hpp>
+#include <util/pprintf.hpp>
+
+namespace nest{
+namespace mc{
+namespace mechanisms {
+namespace gpu {
+
+namespace kernels {
+    __device__
+    inline double atomicAdd(double* address, double val) {
```

Comment:
yes! This needs to be fixed properly at the same time that we adress the issue in the code that is generated by modcc.